scripts/bin/init

```diff
@@ -65,6 +65,7 @@ for entry in ${DROPBEAR_USERS?}; do
 		printf -- '%s\n' '# AUTOGENERATED, DO NOT EDIT!' > "${authorizedKeys:?}"
 		for f in "${authorizedKeys:?}.d/"*; do
 			[ -f "${f:?}" ] || continue
+			printf -- '\n%s\n' "# ${f:?}" >> "${authorizedKeys:?}"
```

Comment:
Instead of appending the file name, I think it would be better to replace `cat` with `awk 1` to ensure that a line break is always printed.

Reply:
Yeah, I'm definitely not skilled enough with `awk` to know what `awk 1` implies.  
Prepending the filename though is just something I like to do whenever concatenating multiple sources, including some kind of information from where a particular block is originating from is always helpful if there's any need of debugging at a later point.